<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1852</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="clock_hour">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="clock_min">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="clock_sec">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="clock_hour_world">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="clock_min_world">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="clock_sec_world">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="intValue" stdset="0">
     <number>99</number>
    </property>
   </widget>
   <widget class="QLabel" name="weather_label">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>181</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="weather_icon">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>270</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="photo_frame_label">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>531</width>
      <height>681</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1852</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
